package.json

```
@@ -12,6 +12,7 @@
     "bigbluebutton-html-plugin-sdk": "0.0.54",
```

I suggest syncing the sdk version with the one currently set in bigbluebutton, since there are breaking changes between `0.0.54` and `0.0.58` that prevent this PR from working correctly with the first.
```suggestion
    "bigbluebutton-html-plugin-sdk": "0.0.58",
```

---

You are completely right! I will update that